BUILDING.md

```diff
@@ -280,8 +280,9 @@ the first command `./configure --coverage`).
 
 _Generating a test coverage report can take several minutes._
 
-To collect coverage for a subset of tests you can set the `CI_JS_SUITES` and
-`CI_NATIVE_SUITES` variables:
+To collect coverage for a subset of tests you can set the CI_JS_SUITES and
+CI_NATIVE_SUITES variables (to run specific suites, e.g., `child-process`, in
```

Comment:
```suggestion
`CI_NATIVE_SUITES` variables (to run specific suites, e.g., `child-process`, in
```

Reply:
whoops, that commit message is wrong, thought you'd fixed which line `and` was on, missed the ticks around `CI_NATIVE_SUITES`; thanks for the review.

Reply:
This one still needs addressing)